tensorboard/pip_package/build_pip_package.sh

```diff
@@ -145,15 +147,39 @@ smoke() {
   curl -fs "http://localhost:$(cat port)/data/logdir" >logdir.json
   grep 'smokedir' logdir.json
   kill $!
+
   # Test TensorBoard APIs
+  export TF_CPP_MIN_LOG_LEVEL=1  # Suppress spammy TF startup logging.
   python -c "
 import tensorboard as tb
 tb.summary.scalar_pb('test', 42)
 from tensorboard.plugins.projector import visualize_embeddings
 from tensorboard.plugins.beholder import Beholder, BeholderHook
 tb.notebook.start  # don't invoke; just check existence
-import tensorboard.summary._tf.summary as tf_summary
 "
+
+  # Exhaustively test various sequences of importing tf.summary.
+  test_tf_summary() {
+    # First argument is subpath to test, e.g. '' or '.compat.v2'.
+    import_attr="import tensorflow as tf; a = tf${1}.summary; a.write; a.scalar"
+    import_as="import tensorflow${1}.summary as b; b.write; b.scalar"
+    import_from="from tensorflow${1} import summary as c; c.write; c.scalar"
+    printf "${import_attr}\n${import_as}\n${import_from}" | python -
```

Comment:
To avoid incorrect behavior in case one of these has a percent sign, you
can use

```sh
    printf '%s\n' "${import_attr}" "${import_as}" "${import_from}" | python -
    printf '%s\n' "${import_attr}" "${import_from}" "${import_as}" | python -
    printf '%s\n' "${import_as}" "${import_attr}" "${import_from}" | python -
    printf '%s\n' "${import_as}" "${import_from}" "${import_attr}" | python -
    printf '%s\n' "${import_from}" "${import_attr}" "${import_as}" | python -
    printf '%s\n' "${import_from}" "${import_as}" "${import_attr}" | python -
```